packages/ui/src/components/Transactions/Transaction.tsx

```diff
@@ -53,7 +54,10 @@ const Transaction = ({
         {!aggregatedData.callData ? (
           <QuestionMarkIcon className="callIcon unknownCall" />
         ) : (
-          <GestureIcon className="callIcon" />
+          <GestureIcon
```

Comment:
There is something wrong with this one, or with the changes done in this file
![image](https://github.com/ChainSafe/Multix/assets/33178835/7eb377e2-e475-4fc4-87ec-f27e38636bee)